index.bs

```diff
@@ -1784,6 +1788,57 @@ endpoint is verified to be a WebTransport endpoint; thus, the Web application
 cannot distinguish between a non-existing endpoint and the endpoint that is not
 willing to accept connections from the Web.
 
+## Authentication using Certificate Hashes {#certificate-hashes}
+
+In the vast majority of cases, a user agent authenticates a TLS connection
+between itself and a remote endpoint by verifying the validity of the TLS
+server certificate provided against the server name in the URL; this approach
+will be further referred to as Web PKI.  The full details of how the Web PKI
+functions are implementation-defined, and go far beyond the scope of this
+docuement; in practice, those tend to be fairly complex, and involve
+procedures for maintaining roots of trust, handling revocations of previously
+issued certificates, as well as defending against key compromise, certificate
+mis-issuance, and numerous other security risks.
+
+This API provides web applications with a capability to connect to a remote
+network endpoint authenticated by the hash of its server certificate, rather
+than its server name.  This enables connections to the endpoints that are not
+capable of getting long-term certificates, including hosts that are ephemeral
+in nature (e.g. short-lived virtual machines), or that are not publicly
+routable.  Since this mechanism substitutes Web PKI-based authentication for an
+individual connection, we need to compare the security properties of both.
+
+Validating against a specific hash has certain desirable security properties
+that Web PKI normally does not provide.  For instance, the hashes used for
+verification are scoped to the context of a single connection, which can limit
+the consequences of any individual certificate being compromised.  In addition,
+hash-based verification has a single unambiguous root of trust (the hash
+provided by the Web application), while the Web PKI allows names to be
+authenticated by multiple certificate authorities;  the latter fact arises from
+a variety of operational and ecosystem concerns that are not usually
+relevant for the use cases where hash-based verification is used.
```

Comment:
"not usually relevant" seems like a value judgment.

Reply:
Removed the paragraph.